### ISSUE TITLE
hr line doesn't show in sections with light background

### ISSUE THREAD

**Issue by githubmonkey** [zjedi/hugo-scroll#61]
This is a very minor issue but I just wanted to let you know that the `<hr>` separator lines show in an odd section (with colored background) but not on an even one. 

I solved it with this small addition in my custom_header.html
``` 
     hr { border-top: 1px solid #888888; }
     .odd hr { border-top: 1px solid #efefef; }
````

**Comment by zjedi** [comment]
will probably be fixed in #4

**Comment by zjedi** [comment]
@githubmonkey now fixed at master
